### PR TITLE
WIP: Remove config-tracing, since already there via eventing

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- Remove empty `config-tracing`, which is already there via eventing core, so would override that (potentially)

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: hack/update-checksums.sh [hunk 1]
```diff
@@ -26,4 +26,4 @@ fi
 
 source $(dirname $0)/../vendor/knative.dev/hack/library.sh
 
-go run "${REPO_ROOT_DIR}/vendor/knative.dev/pkg/configmap/hash-gen" "${REPO_ROOT_DIR}"/control-plane/config/100-config-tracing.yaml
+#go run "${REPO_ROOT_DIR}/vendor/knative.dev/pkg/configmap/hash-gen" "${REPO_ROOT_DIR}"/control-plane/config/100-config-tracing.yaml
```
Comment (matzew) [inline hack/update-checksums.sh:29]: I think we can either remove the file or just remove this line

**Comment by matzew** [comment]
@pierDipi Since this file gets installed via `eventing`, not sure we need to have the duplicate on ours here

**Comment by matzew** [comment]
Ok, so I'd think we fine-tweak the actual yaml that get's produced? 

E.g. broker VS sink

**Comment by pierDipi** [comment]
Yes, we're fine as long as `config-tracing` is in the system namespace (the data plane doesn't start without config-tracing in the right place)

**Pull request closed without merging**